### PR TITLE
feat(android): implement Ti.UI.View.borderRadiusCorners

### PR DESCRIPTION
**Pull request by garymathews**
- Continuation of https://github.com/appcelerator/titanium_mobile/pull/11114 from @m1ga
- Implement `Ti.UI.View.borderRadiusCorners`

##### TEST CASE
```JS
const window = Ti.UI.createWindow({ backgroundColor: 'white' });
const view_a = Ti.UI.createView({
	backgroundColor: 'red',
	width: 100,
	height: 50,
	top: 100,
	borderRadius: 20,
	borderRadiusCorners: [
		Ti.UI.BORDER_CORNER_TOP_LEFT,
		Ti.UI.BORDER_CORNER_TOP_RIGHT
	]
});
const view_b = Ti.UI.createView({
	backgroundColor: 'blue',
	width: 100,
	height: 50,
	top: 150,
	borderRadius: 20,
	borderRadiusCorners: [
		Ti.UI.BORDER_CORNER_BOTTOM_LEFT,
		Ti.UI.BORDER_CORNER_BOTTOM_RIGHT
	]
});

window.add([ view_a, view_b ]);
window.open();
```
- Should see a blue and red square with rounded corners

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27946)

### CODE REVIEW

File: tests/Resources/ti.ui.view.addontest.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.View', () => {
+
+	it.android('borderRadiusEdges', () => {
```
Comment (vijaysingh-axway) [inline tests/Resources/ti.ui.view.addontest.js:14]:
```suggestion
	it.android('borderRadiusCorners', () => {
```

File: tests/Resources/ti.ui.view.addontest.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		const view = Ti.UI.createView({
+			width: 100,
+			height: 100,
+			borderRadiusCorners: [
```
Comment (vijaysingh-axway) [inline tests/Resources/ti.ui.view.addontest.js:18]: Shouldn't this give an exception as this property is valid with 'Titanium.UI.View.borderRadius'.

Comment (garymathews) [reply tests/Resources/ti.ui.view.addontest.js:18]: I don't think it should, we don't when setting `borderWidth`; this is just a test to see if the properties get set.

File: apidoc/Titanium/UI/View.yml [hunk 3]
```diff
@@ -1323,6 +1323,13 @@ properties:
     type: Number
     default: 0
 
+  - name: borderRadiusCorners
+    summary: An array of corners to use specified <Titanium.UI.View.borderRadius> on.
+    type: Array<Number>
+    constants: Titanium.UI.BORDER_CORNER_*
+    platforms: [android, iphone, ipad]
+    since: "9.1.0"
+
```
Comment (jquick-axway) [inline apidoc/Titanium/UI/View.yml:1332]: This feature is only supported on Android 5.0 and above, right?
In which case we need to add the following...
`osver: {android: {min: "5.0"}}`

Comment (jquick-axway) [reply apidoc/Titanium/UI/View.yml:1332]: We should also document that if this property is "undefined", then `borderRadius` is applied to all corners. If an array is assigned to this property, then it must specify each corner to apply to radius to. If assigned an empty array, then `borderRadius` is not applied to any other corners.

**Comment by vijaysingh-axway** [comment]
@garymathews Need to update implementation as per given suggestion https://github.com/appcelerator/titanium_mobile/pull/11770#issuecomment-648510542 .

**Comment by garymathews** [comment]
Closing, superseded by https://github.com/appcelerator/titanium_mobile/pull/11796

**Pull request closed without merging**